arangod/MMFiles/MMFilesRestReplicationHandler.cpp

```diff
@@ -589,7 +597,13 @@ void MMFilesRestReplicationHandler::handleCommandInventory() {
     DatabaseFeature::DATABASE->inventory(builder, tick, nameFilter);
   } else {
     // add collections and views
-    ExecContextSuperuserScope scope(ExecContext::current().isAdminUser());
```

Comment:
@graetzer @jsteemann what is protected by this? The _users collection? The _wal? Why is a new scope needed?